docs/decisions/0005-project-labels.md

```diff
@@ -0,0 +1,154 @@
+# Context and Problem Statement
+
+The goal is to put into place standard labels across all SUSE Rancher projects,
+which allows people from multiple spheres of interest to have a single glance
+into top-level work.
+
+Issues and labels are used for:
+
+- Tracking defects
+- Tracking feature requests
+- Planning releases
+- Managing workflows
+- Reporting stats
+
+The introduction of these changes should have minimum impact on our current
+workflow since we are mainly renaming already existing labels. We use custom
+fields to define [epics][1] & [milestones][2]. We will carry on using them,
+together with the `kind/epic` label, since they are a great way of organizing
+the work.
+
+## Considered Options
+
+Labels can be found here: [https://github.com/aquarist-labs/s3gw/labels][3]
+
+### Issue conventions
+
+All issues should be actionable (within a reasonable timeframe). Usage questions
+and design brainstorming should happen in GitHub discussions.
+
+### Titles
+
+Issue titles should be short (up to 70 characters), with the most important info
+at the front.
+
+### Labels
+
+Label names are simple strings without spaces or special characters (including
+emojis) to make it easy to type them into GitHub queries.
+
+Labels are grouped into orthogonal "categories", and ideally, an issue should
+have no more than one label from each category, e.g. `kind/bug`,
+`area/kubernetes`, `component/ui`.
+
+When an issue seems to require multiple labels from the same category, it is
+often a hint that it should be split into multiple issues.
+
+The following list aims to comprehensively document all our labels.
+
+### `area/*`
+
+The `area` category species is the user's visible area that is affected by the
+issue. It helps to find a duplicate issue during reporting or the triage
+process.
+
+An example is `area/kubernetes`.
+
+### `component/*`
+
+The `component/*` category specifies the part of the implementation that will
+need to be modified during the implementation.
+
+An example is `component/ui`.
+
+### `kind/*`
+
+This is the "document kind". Each issue **must** have exactly one `kind/*`
+label. The labels `kind/bug` and `kind/enhancement` are being standardized
+across SUSE Rancher projects.
+
+#### `kind/bug`
+
+This issue describes a defect in the software. Sometimes it requires a judgement
+call, but in general, incomplete functionality, or different behaviour from a
+similar project should be labelled as `kind/enhancement` instead.
+
+#### `kind/enhancement`
+
+This issue requests additional (or a change in) functionality. The request
+typically comes from a user or the PO.
+
+#### `kind/epic`
+
+An epic is an umbrella issue to implement a feature that can span over one or
+many release cycle(s). It includes a work breakdown as a task list for
```

Comment:
s/cycle(s)/cycles/

As is, given the phrase, it makes perfect sense to use 'cycles'.

Reply:
fixed!